docs/wiki/PipelinesDesign.md

```diff
@@ -6,34 +6,44 @@ This section gives you an overview of the design principals the pipelines follow
 
 ### _Navigation_
 
+- [Pipelines Design](#pipelines-design)
```

Comment:
I think there's some autoformatting coming from a VSC extension. 

- Table of contents: Navigation and titles should be removed. 
- Tables: open question. I don't have a strong opinion but I think we avoided to format tables at least for readme files for readability.

Reply:
reverted to original TOC